project/frontend/src/components/ChartCard/ChartCard.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import Chart from "react-google-charts";
+import classes from "./ChartCard.module.css";
+
+
+const ChartCard = (props) => {
+    return (
+        <div className={classes.ChartContainer}>
+            <Chart
+                chartType="LineChart"
+                loader={<div>Loading Chart</div>}
+                data={[ // hard coded for now
+                    ['x', 'trend 1', 'trend 2', 'trend 3'],
+                    [0, 1, 2, 90],
+                    [1, 0, 6, 80],
+                    [2, 34, 23, 23],
+                    [3, 22, 43, 12],
+                    [4, 34, 56, 91]
+                ]}
+                options={{
+                    hAxis : {
+                        title : 'Date'
```

Comment:
Can you change 'Date' and 'Popularity' to props.haxis and props.vaxis or something so this can be passed in later

Reply:
> Just want to check is the size of the chart dynamic? Does the chart expand to fill the space it's in?

I'm really not too sure the best way of doing that, I tend to solve all my css problems with hard coded padding. Is this something stylistic you'd prefer to have a play with yourself?

Reply:
> Can you change 'Date' and 'Popularity' to props.haxis and props.vaxis or something so this can be passed in later

fixed